client/elements/addons/sc-top-sheet-search-filter.js

```diff
@@ -134,40 +139,42 @@ export class SCTopSheetSearchFilter extends SCTopSheetCommon {
             <tr>
               <td>Search <code>in</code> a specific collection with SuttaCentral IDs.</td>
               <td>in:dn cat<br>
-                  in:an dog<br>
-                  in:an Buddha OR Sāvatthī<br>
-                  in:an Buddha AND Sāvatthī
+                  in:an4 dog<br>
+                  in:sutta moat</br>
+                  in:abhidhamma feelings</br>
+                  in:kn flame</br>
+                  in:an Buddha OR Sāvatthī
               </td>
             </tr>
 
             <tr>
               <th colspan="2">ebt search</th>
             </tr>
             <tr>
-              <td>Narrow search to “Early Buddhist Texts” (<code>ebt</code>). This is a shortcut and not a definitive list of what is early. Equivalent to the following.<br>
-                <code>in:dn, da, mn, ma, sn, sa, sa-2, sa-3, an, ea, ea-2, kp, iti, ud, snp,
+              <td>Narrow search to “Early Buddhist Texts” (<code>ebt</code>). This is a shortcut and not a definitive list of what is early. Equivalent to the following:<br>
```

Comment:
We agreed not to use the word `equivalent` no?

Reply:
Right we did!